tests/test_selectors.py

```diff
@@ -74,6 +66,16 @@ def test_categorical(request: Any, constructor: Any) -> None:
     compare_dicts(result, expected)
 
 
+def test_dask_categorical() -> None:
```

Comment:
Hack to test categorical selector until `cast` is not implemented

Reply:
No worries :) 2054694f09d70bbffc59a3390f4feaabb465f76b